x-pack/plugins/observability_solution/slo/public/embeddable/slo/alerts/slo_alerts_embeddable_factory.tsx

```diff
@@ -143,18 +151,28 @@ export function getAlertsEmbeddableFactory(
                   kibanaVersion,
                 }}
               >
-                <Router history={history}>
-                  <QueryClientProvider client={queryClient}>
-                    <SloAlertsWrapper
-                      onEdit={onEdit}
-                      deps={deps}
-                      slos={slos}
-                      timeRange={fetchContext.timeRange ?? { from: 'now-15m/m', to: 'now' }}
-                      reloadSubject={reload$}
-                      showAllGroupByInstances={showAllGroupByInstances}
-                    />
-                  </QueryClientProvider>
-                </Router>
+                <PluginContext.Provider
```

Comment:
Linking with this comment [here](https://github.com/elastic/kibana/pull/182094#discussion_r1615751360). There is indeed an `SloEmbeddableContext` which I introduced as part of refactoring the alerts embeddable and I wanted to avoid. I will create an issue for this